.github/labeler.yml

```diff
@@ -0,0 +1,63 @@
+"Area: boards":
+  - boards/**/*
+
+"Area: build system":
+  - Makefile.*
+  - makefiles/**/*
+
+"Area: CI":
+  - .circleci/**/*
+  - .github/**/*
+  - .murdock
+
+"Area: core":
+  - core/**/*
+
+"Area: cpu":
+  - cpu/**/*
+
+"Area: doc":
+  - ./**/*.txt
+  - doc/**/*
+  - CONTRIBUTING.md
+  - MAINTAINING.md
+  - README.md
+
+"Area: drivers":
+  - drivers/**/*
+
+"Area: examples":
+  - examples/**/*
+
+"Area: network":
+  - sys/net/**/*
+  - sys/include/net/**/*
```

Comment:
This could also include all the `pkg` implementing network stacks (`lwip`, `openthread`, `openwsn`, ...) and drivers implementing network devices.